### ISSUE TITLE
Does not check dev dependencies

### ISSUE THREAD

**Issue by jdolle** [salimkayabasi/types-checker#12]
I have `enzyme` installed as a devDependency and this doesn't catch that I need to install `@types/enzyme`.

**Comment by salimkayabasi** [comment]
hey @jdolle,

the main purpose is catching all `dependencies` but I can add one more flag to check `devDependencies` also.

**Comment by salimkayabasi** [comment]
plz check the version `0.0.8`.

BTW, watch out `@types/typescript` module because it is not a correct suggestion
u can exclude it by using `package.json` file
```json
# package.json
...
 "tscheck": {
    "exclude": [
      "typescript"
    ]
  },
...
```

**Comment by jdolle** [comment]
Nice! Thank you very much.
